components/ResultsPage/BenefitCard.tsx

```diff
@@ -71,13 +77,13 @@ export const BenefitCard: React.VFC<{
 
       <div className="mt-4">
         {links &&
-          links.map(({ text, url, icon, alt }, index) => (
+          links.map(({ text, url, icon, alt, idSuffix }, index) => (
             <div key={index} className="flex items-center py-4 text-content">
               <div>
                 <Image src={`/${icon}.png`} alt={alt} width="30" height="44" />
               </div>
               <div className="pl-5 w-full">
-                <DSLink id={`link${index}`} href={url} text={text} />
+                <DSLink id={`${idSuffix}Link${index}`} href={url} text={text} />
```

Comment:
suggestion use Link${benefitKey}-${index}  instead, so idSuffix is not only not required, but no other change is required.

Reply:
good idea